lua/weapons/gmod_tool/stools/starfall_component.lua

```diff
@@ -13,10 +13,12 @@ TOOL.ClientConVar["Type"] = "1"
 TOOL.ClientConVar["parent"] = "1"
 cleanup.Register("starfall_components")
 
+local MakeComponent
```

Comment:
I dont like outscoping variables this way if they aren't shared. Can we at least add a comment that it's serverside only?

Reply:
Well then edit starfall_preprocessor while you're at it, because it's doing the same exact thing